### PR TITLE
Fix path for swagger-ui

### PR DESCRIPTION
**Pull request by satnam6502**

### CODE REVIEW

**Comment by a-robinson** [comment]
LGTM

File: docs/api.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Primary system and API concepts are documented in the [User guide](user-guide.md
 
 Overall API conventions are described in the [API conventions doc](api-conventions.md).
 
-Complete API details are documented via [Swagger](http://swagger.io/). The Kubernetes apiserver (aka "master") exports an API that can be used to retrieve the [Swagger spec](https://github.com/swagger-api/swagger-spec/tree/master/schemas/v1.2) for the Kubernetes API, by default at `/swaggerapi`, and a UI you can use to browse the API documentation at `/swaggerui`. We also periodically update a [statically generated UI](http://kubernetes.io/third_party/swagger-ui/).
+Complete API details are documented via [Swagger](http://swagger.io/). The Kubernetes apiserver (aka "master") exports an API that can be used to retrieve the [Swagger spec](https://github.com/swagger-api/swagger-spec/tree/master/schemas/v1.2) for the Kubernetes API, by default at `/swaggerapi`, and a UI you can use to browse the API documentation at `/swagger-ui`. We also periodically update a [statically generated UI](http://kubernetes.io/third_party/swagger-ui/).
```
Comment (bgrant0607) [inline docs/api.md]: Why was this changed? Did you try this URL on your master?

You can see the path here:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/master/master.go#L560

**Comment by a-robinson** [comment]
This is what I see on a v0.15.0 cluster:
```
arob@k8s-v15-master:~$ curl localhost:8080
{
  "paths": [
    "/api",
    "/api/v1beta1",
    "/api/v1beta2",
    "/api/v1beta3",
    "/healthz",
    "/healthz/ping",
    "/logs/",
    "/metrics",
    "/static/",
    "/swagger-ui/",
    "/swaggerapi/",
    "/validate",
    "/version"
  ]
}arob@k8s-v15-master:~$ curl localhost:8080/swaggerui
<a href="/swaggerui/">Moved Permanently</a>.

arob@k8s-v15-master:~$
```

**Comment by satnam6502** [comment]
I changed `/swaggerui` (wrong) to `/swagger-ui` (right).

**Comment by bgrant0607** [comment]
Ugh. Thanks. That's inconsistent with the line I cited, which should be changed to match or removed if unnecessary.

https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/ui/installsupport.go#L45
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/master/master.go#L560

@nikhiljindal
